src/index.ts

```diff
@@ -22,8 +22,21 @@ interface TelescopeNode {
   telescopicOut: TelescopicOutput;
 }
 
+interface Config {
+  separator?: string;
+  hoverable?: boolean;
+}
+
+const _DEFAULT_CONFIG: Config = {
```

Comment:
nit: prefer upper camel for global configs so `DefaultConfig` also for some reason I had trouble with getting these global constants to be defined when importing (which is why I had to put the others like `BulletSeparators` into the function. 

Just to make sure, did you validate this all flows properly when importing from lib file locally?

Reply:
Yes - it works for me locally!